contrib/forgejo/files/forgejo

```diff
@@ -0,0 +1,8 @@
+type = process
+command = /usr/bin/forgejo web
```

Comment:
i've seen other distros pass `-c /etc/forgejo/app.ini` here but that doesn't seem necessary at all

also might want to depend on postgres here

Reply:
it can use any db including sqlite so not sure why it would depend on postgres

Reply:
most you can put without also installing postgres is after =

Reply:
i was using the term "depend" loosely here, i was thinking more about `after` like you mention